chart/values.yaml

```diff
@@ -141,6 +141,14 @@ controller:
   # This value defines additional annotations for the deployment. This should be formatted as a YAML object (map)
   annotations: {}
 
+  # Controller pre-delete hook labels and annotations
```

Comment:
It would be great if we could move the hooks configuration to https://github.com/hashicorp/vault-secrets-operator/blob/67fce39ae4a4929a712211e6a5908fbf5fa14236/chart/values.yaml#L786 .